tx_input_tests.json

```diff
@@ -14,21 +14,6 @@
    }
   ]
  },
- {
-  "description": "When the input is an SLP-invalid BCH-only tx, the GENESIS tx should be SLP-invalid since it is not token type 1 or 129 (NFT1 parent).",
```

Comment:
This test was intended to cover cases where a different token version's GENESIS relies on more than just the parsing to be a valid genesis (e.g., NFT1 child).

But thinking about it again, I agree it should be removed because the parser is throwing.  Also, other token consensus rules will need a whole other set of unit tests.